templates/package/view.tmpl

```diff
@@ -39,7 +39,7 @@
 							{{if .HasRepositoryAccess}}
 							<div class="item">{{svg "octicon-repo" 16 "mr-3"}} <a href="{{.PackageDescriptor.Repository.HTMLURL}}">{{.PackageDescriptor.Repository.FullName}}</a></div>
 							{{end}}
-							<div class="item">{{svg "octicon-calendar" 16 "mr-3"}} {{.PackageDescriptor.Version.CreatedUnix.FormatDate}}</div>
+							<div class="item time">{{svg "octicon-calendar" 16 "mr-3"}} {{TimeSinceUnix .PackageDescriptor.Version.CreatedUnix $.locale}}</div>
```

Comment:
What's the purpose of the `time` class?

Reply:
I would have guessed so that the tooltip will be shown?
I don't think `TimeSinceUnix` creates that.

Reply:
oh. I thought it simply converts the number to a human-readable string…
Then why do I find a few of those instances where the declaring tag has `class="time">{{TimeSinceUnix …}}`?

Reply:
Apparently there is a `time` style class.

Reply:
Those `.time` selectors are not matching here I think.

BTW, I recommend `rg` over `ag`, it honors `.ignore` in this case, as opposed to `ag` which doesn't 😉

Reply:
I thought the class was necessary too...